### PR TITLE
Reserved tags for specs copied to support.

### PR DESCRIPTION
**Pull request by mgritter**

### CODE REVIEW

File: tags/reserved_tags.go [hunk 1]
```diff
@@ -147,6 +147,19 @@ const (
 	XAkitaDumpFilterFlag Key = "x-akita-dump-filter-flag"
 )
 
+// Tags applied to a copy of the spec
+const (
+	XAkitaOriginalOrganization = "x-akita-original-organization"
```
Comment (liujed) [inline tags/reserved_tags.go:152]: Is this an ID or a name?

File: tags/reserved_tags.go [hunk 2]
```diff
@@ -147,6 +147,19 @@ const (
...
+
+	XAkitaOriginalService = "x-akita-original-service"
```
Comment (liujed) [inline tags/reserved_tags.go:154]: `XAkitaOriginalServiceName = "x-akita-original-service-name"`

File: tags/reserved_tags.go [hunk 3]
```diff
@@ -147,6 +147,19 @@ const (
...
+
+	XAkitaOriginalServiceID = "x-akita-original-service-id"
+
+	XAkitaOriginalName = "x-akita-original-name"
```
Comment (liujed) [inline tags/reserved_tags.go:158]: `XAkitaOriginalSpecName = "x-akita-original-spec-name"`

**Review by liujed — APPROVED** [review]
Nits for some slightly less ambiguous tag names.

**Review by thatplguy — APPROVED** [review]

File: tags/reserved_tags.go (hunk 1 above)
Comment (mgritter) [reply tags/reserved_tags.go]: Changed to use ID suffix to be consistent.

File: tags/reserved_tags.go (hunk 3 above)
Comment (mgritter) [reply tags/reserved_tags.go]: I changed to x-akita-original spec; original-service and original-spec are names; original-service-id and original-spec-id are IDs.

**Pull request merged**